### PR TITLE
Add information about how to install frontend dependencies

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+```
+brew install node
+```
+
+### Frontend dependencies
+```
+npm install
+```
+
 ## Development server
```
Comment (nakhbari) [inline web/README.md]: Do you mind also adding a build prod with the command `npm run build`?

Comment (KrauseFx) [reply web/README.md]: Done 👍

**Comment by KrauseFx** [comment]
Let me know if that's how you imagined, if not, feel free to directly edit the PR using the GitHub UI 👍

**Comment by nakhbari** [comment]
🚢 it!

**Pull request merged**